integrations/segmentio/lib/index.js

```diff
@@ -331,16 +332,19 @@ Segment.prototype.normalize = function(message) {
       }
 
       for (var j = 0; j < maybeBundledConfigIds[name].length; j++) {
+        bundledIdsComputedAtRuntime = true;
         var id = maybeBundledConfigIds[name][j]
         bundledConfigIds.push(id)
       }
     }
 
-
     msg._metadata = msg._metadata || {};
     msg._metadata.bundled = bundled;
     msg._metadata.unbundled = this.options.unbundledIntegrations;
     msg._metadata.bundledIds = bundledConfigIds;
+    msg._metadata.maybeBundledConfigIds = maybeBundledConfigIds;
+    msg._metadata.bundledIdsComputedAtRuntime = bundledIdsComputedAtRuntime;
+    msg._metadata.addBundledMetadata = this.options.addBundledMetadata;
```

Comment:
Maybe I missed this before - what's this option?

Reply:
Ah, I see here: https://github.com/segmentio/ajs-renderer/blob/63f438e1f1a5f2694f7a0d91f26f3e0a88287d4b/src/builder/settings-by-name.ts#L97